internal/core/validate.go

```diff
@@ -118,6 +118,16 @@ func ValidateSecretKey() ArgSpecValidateFunc {
 }
 
 func ValidateOrganisationID() ArgSpecValidateFunc {
+	return func(argSpec *ArgSpec, valueI interface{}) error {
+		value := valueI.(string)
+		if value == "" {
+			return nil
+		}
+		return ValidateOrganisationIDRequired()(argSpec, valueI)
+	}
+}
+
+func ValidateOrganisationIDRequired() ArgSpecValidateFunc {
```

Comment:
can you comment: why is this so complicated to validate organization ID? Why do we need 2 functions? Can't we just validate a UUID?